.travis/run.sh

```diff
@@ -1,11 +0,0 @@
-#!/bin/bash
-
-set -e
-set -x
-
-if [[ $TRAVIS_PYTHON_VERSION == pypy ]]; then
-    py.test --hypothesis-profile travis test/
```

Comment:
Have we lost the Hypothesis profile on Travis? That seems like it would be a regression &ndash; or is it sneaking in somewhere I can't see?